packages/layout/src/components/SideNav/MedlySidenavHeader/MedlySidenavHeader.styled.tsx

```diff
@@ -6,21 +6,10 @@ export const SidenavHeader = styled('div')<HeaderStyledProps>`
     overflow: hidden;
     position: relative;
     min-height: ${({ theme }) => theme.sideNav.closeSize};
+    padding: 0 1.9rem;
+    box-sizing: border-box;
+    display: flex;
     align-items: center;
-    justify-items: center;
-    display: grid;
-    grid-template-columns: ${({
-        theme: {
-            sideNav: { closeSize, openSize }
-        }
-    }) => `${closeSize} calc(${openSize} - ${closeSize}) `};
-
-    svg:last-child {
-        justify-self: left;
-        transform: translateX(-1rem);
-        transition: opacity 200ms;
-        opacity: ${({ isHovered, isExpanded }) => (isHovered || isExpanded ? 1 : 0)};
```

Comment:
We need to add this transition effect in `Logo` component or leave it in this component.

Reply:
Done.